app/models/user.rb

```diff
@@ -1,9 +1,21 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  AVAILABLE_LOCALES = (I18n.available_locales.map(&:to_s) << nil).freeze
```

Comment:
```suggestion
  AVAILABLE_LOCALES = I18n.available_locales.map(&:to_s).freeze
```

No needs of `nil`. You have to use `allow_nil: true` in validator (apply the same on every validators related to settings